shoplist/src/main/java/com/netguru/codereview/network/model/ShopListResponse.kt

```diff
@@ -0,0 +1,10 @@
+package com.netguru.codereview.network.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+data class ShopListResponse(
+    val list_id: String,
```

Comment:
here list_id must be listId
please follow the camel case for local variable, to overcome this issue you can use @SerializedName("list_id") Gson Annotation to specify the variables comes from the api side.